src/nbl/video/ILogicalDevice.cpp

```diff
@@ -6,6 +6,17 @@ using namespace nbl::video;
 
 E_API_TYPE ILogicalDevice::getAPIType() const { return m_physicalDevice->getAPIType(); }
 
+void ILogicalDevice::addJITIncludeLoader()
+{
+    if(auto cc = (m_physicalDevice && m_compilerSet) ? m_compilerSet->getShaderCompiler(asset::IShader::E_CONTENT_TYPE::ECT_HLSL) : nullptr)
+    {
+        if(auto finder = cc->getDefaultIncludeFinder())
+        {
+            finder->addSearchPath("", core::make_smart_refctd_ptr<CJITIncludeLoader>(m_physicalDevice->getLimits(), m_physicalDevice->getFeatures()));
```

Comment:
you want the search path to be `nbl/builtin/hlsl/jit` not empty

Reply:
fixing